lib/ui/views/projects/project_details_view.dart

```diff
@@ -75,18 +76,40 @@ class _ProjectDetailsViewState extends State<ProjectDetailsView> {
         color: Colors.white,
       ),
       child: ClipRRect(
-        child: PhotoView.customChild(
-          backgroundDecoration: BoxDecoration(
-            color: Colors.white,
-          ),
-          initialScale: 1.0,
-          child: FadeInImage.memoryNetwork(
-            height: 100,
-            width: 50,
-            placeholder: kTransparentImage,
-            image:
-                '${EnvironmentConfig.CV_API_BASE_URL.substring(0, EnvironmentConfig.CV_API_BASE_URL.length - 7) + _recievedProject.attributes.imagePreview.url}',
-          ),
+        child: Stack(
+          alignment: Alignment.bottomRight,
+          children: [
+            PhotoView.customChild(
+              backgroundDecoration: BoxDecoration(
+                color: Colors.white,
+              ),
+              initialScale: 1.0,
+              child: FadeInImage.memoryNetwork(
+                height: 100,
+                width: 50,
+                placeholder: kTransparentImage,
+                image:
+                    '${EnvironmentConfig.CV_API_BASE_URL.substring(0, EnvironmentConfig.CV_API_BASE_URL.length - 7) + _recievedProject.attributes.imagePreview.url}',
+              ),
+            ),
+            Material(
+              color: CVTheme.primaryColor,
+              child: IconButton(
+                onPressed: () {
+                  Navigator.push(
```

Comment:
This works but I will encourage you to use [GetX's Get.toNamed](https://pub.dev/packages/get).